src/components/RecipeItem/RecipeItem.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { AiTwotoneHeart } from "react-icons/ai";
+
+const RecipeItem = ({ recipeName, recipeImage, recipeURI }) => {
+  return (
+    <div className=" item recipe-Item items-center hover:scale-110 mb-12">
```

Comment:
There is a space before the first className. I see that you have some random spaces here and there, might be nice to use something like `Headwind` (vs code plugin) to order your CSS classNames.